ext/standard/config.m4

```diff
@@ -303,14 +303,6 @@ if test "$ac_cv_attribute_aligned" = "yes"; then
   AC_DEFINE([HAVE_ATTRIBUTE_ALIGNED], 1, [whether the compiler supports __attribute__ ((__aligned__))])
 fi
 
-dnl
-dnl Check for available functions
-dnl
-dnl log2 could be used to improve the log function, however it requires C99. The
-dnl check for log2 should be turned on, as soon as we support C99.
-AC_CHECK_FUNCS(asinh acosh atanh log1p hypot)
-AC_FUNC_FNMATCH
```

Comment:
I'm assuming the removal of AC_FUNC_FNMATCH here was not intentional?

Reply:
Ah yes indeed